app/controllers/publish/providers/school_search_controller.rb

```diff
@@ -5,7 +5,15 @@ module Providers
     class SchoolSearchController < PublishController
       helper_method :query, :search_result_title_component
 
-      before_action :authorize_provider
+      before_action :authorize_provider, except: %i[suggest]
+
+      def suggest
```

Comment:
I think we've decided to avoid adding custom methods in controllers and keep the actions as RESTful as possible. I'd opt we introduce a new api controller specifically for this like:

https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/controllers/api/schools_controller.rb